Week1/homework/Exercise_5_The cat walk/cat.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>Cat Walk</title>
+  </head>
+  <body>
+    <img style="position:absolute;" src="http://www.anniemation.com/clip_art/images/cat-walk.gif" />
+    
+    <script>
+    //let img1 = document.getElementsByTagName('img').item[0];
+    let img1 = document.querySelector('img');    
+    img1.style.left = '0px';
+ 
+   function moveCat() {
+     var x = window.innerWidth;
+     console.log(x /2 );
```

Comment:
console.logs should be removed before committing the code. If you want, you can add some comments to your code